src/scope.jl

```diff
@@ -137,4 +140,11 @@ function scopes(x::EXPR, state)
         end
     end
     return s0
-end
\ No newline at end of file
+end
+
+# Add an `eval` method
+function add_eval_method(x, state)
```

Comment:
I think that we should techinically add `include` here as well, at least for modules.

Reply:
Thanks - you're right however I think I'll leave that one for the moment